frontend/src/pages/ActivityReport/Pages/Review/Approver/index.js

```diff
@@ -16,9 +16,18 @@ const Approver = ({
   children,
   error,
 }) => {
-  const { managerNotes, additionalNotes, status } = formData;
+  const {
+    managerNotes, additionalNotes, status, approvingManager,
+  } = formData;
   const review = status === REPORT_STATUSES.SUBMITTED || status === REPORT_STATUSES.NEEDS_ACTION;
   const approved = status === REPORT_STATUSES.APPROVED;
+  const [approverStatusList, updateApproverStatusList] = useState([]);
+
+  useEffect(() => {
+    if (formData && approvingManager && status) {
+      updateApproverStatusList([{ approver: approvingManager.fullName, status }]);
+    }
+  }, [formData, approvingManager, status]);
```

Comment:
why do we need the `useEffect` here? Are `approvingManager` and `status` sometimes null? If we can remove the `useEffect` we can also simplify the approval status state into a simple variable.

Reply:
The idea here was that we have the correct list of approving managers when the page loads or if the selected approving manager changes on the page.

If you think that a simple variable will have the same effect I will change. Let me know your thouhts.

Reply:
Scratch my last comment you are correct. The approver cannot change the approving managers from this path. :-)